src/partials/connectors-tablerow.hbs

```diff
@@ -0,0 +1,51 @@
+<tr class="tr" tabindex="0">
+  <td class="td">
+    <img class="img" src="/_/img/icons/connectors/icon-amazon-s3.svg" width="30" alt="Amazon S3 logo">
```

Comment:
@runnabro Do we really want a logo per connector, or should we use the vendor logo?

Reply:
That's my personal preference—but technically the logo is what appears in Exchange. I'll raise the issue with the team and see what they think.

Reply:
@runnabro @mojavelinux I would suggest using the vendor logo also. Not sure why we have different logos in Exchange. I'll follow up with Meera (the connectors PM).

Reply:
Actually, ignore my previous comment. Meeta just mentioned that we cannot use some of the vendors icons because of licensing issues. The Exchange team has dealt with this issue in the past. As long as we use the icon that is in Exchange, we should be ok.

Reply:
To be clear, I'm not concerned with whether its a logo or icon or png or svg. What I'm looking for is a consistent pattern so that we can compute the path to it from the metadata. I'm also curious whether the logo is that of the vendor (which would need to be defined in the metadata) or that of the connector (which can be based on the component name).

Reply:
I think Steven is saying that they prefer the connector logo.